src/app/courses/course/duration.pipe.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { DurationPipe } from './duration.pipe';
+
+describe('DurationPipe', () => {
+  const pipe = new DurationPipe();
```

Comment:
What does your pipe do? It represents time formatting passed number to a string.
What if I pass falsy value or not a number?

Reply:
done